Dockerfile-multistage

```diff
@@ -7,7 +7,7 @@ ENV     CHEMISTRY_PACKAGE=${CHEMISTRY_PACKAGE}
 
 FROM 	dependencies as jchem
 COPY 	lib/jchem-16.4.25.0.jar /lib/jchem-16.4.25.0.jar
-RUN   mvn install:install-file -Dfile=/lib/jchem-16.4.25.0.jar -DartifactId=jchem -DgroupId=com.chemaxon -Dversion=16.4.25.0 -Dpackaging=jar -DgeneratePom=true -DcreateChecksum=true
+RUN 	--mount=type=cache,target=/root/.m2  mvn install:install-file -Dfile=/lib/jchem-16.4.25.0.jar -DartifactId=jchem -DgroupId=com.chemaxon -Dversion=16.4.25.0 -Dpackaging=jar -DgeneratePom=true -DcreateChecksum=true
```

Comment:
@philippcheung This will likely cause a conflict when you merge your branch but it should fix that issue where you had to comment out the cache to get the jchem builds working.